docs/adding-a-new-app.md

```diff
@@ -0,0 +1,125 @@
+# Adding a new app to GOV.UK
+
+This Puppet repository defines all the apps which run on GOV.UK.
+
+Create a new file in `modules/govuk/manifests/apps` named `my_app.pp`:
+
+```
+# Be sure to include documentation of the class at the top of the file, as
+# per https://docs.puppetlabs.com/guides/style_guide.html#puppet-doc.
+#
+# You may wish to copy an existing app, eg. calculators:
+# https://github.com/alphagov/govuk-puppet/blob/3e926f0d/modules/govuk/manifests/apps/calculators.pp
+#
```

Comment:
Can we add a template here to make it easier for people to copy-paste?

Example:

```puppet
# == Class: govuk::apps::my_app
#
# Full description of class govuk::apps::my_app here
#
# === Parameters
#
# Document parameters here.
#
# [*parameter1*]
#   Explanation of what this parameter affects and what it defaults to.
#   e.g. "Specify one or more upstream ntp servers as an array."
#
```